server/services/reporting.js

```diff
@@ -142,4 +142,29 @@ const getHiredParticipantsReport = async () => {
   }));
 };
 
-module.exports = { getReport, getParticipantsReport, getHiredParticipantsReport };
+const getRejectedParticipantsReport = async () => {
+  const rejectedEntries = await dbClient.db[collections.PARTICIPANTS_STATUS].join({
+    participantJoin: {
+      type: 'LEFT OUTER',
+      relation: collections.PARTICIPANTS,
+      on: {
+        id: 'participant_id',
+      },
+    },
+  }).find({
+    current: true,
+    status: 'rejected',
+    // 'employerSiteJoin.body.siteId::int >': 0, // Ensures that at least one site is found
```

Comment:
We can definitely remove this line, this was only relevant for the hired report since that's the point at which the employer enters their site :)